code/sorting/cycle_sort/cycle_sort_by_SKAL.cpp

```diff
@@ -0,0 +1,83 @@
+// C++ program to impleament cycle sort
+#include <iostream>
+using namespace std;
+ 
+// Function sort the array using Cycle sort
+void cycleSort (int arr[], int n)
```

Comment:
Since this is C++, you should probably use `vector`s, since they are less error prone than normal arrays, and don't require the size to be passed as a parameter.